rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -999,6 +999,69 @@ private Integer getProposedFederationFormatVersionFromRepository() {
                 .map(BridgeSerializationUtils::deserializeInteger)
                 .orElse(null);
         }
+
+        @Test
+        void getProposedFederation_whenThereIsNoProposedFederationSavedNorSet_shouldReturnEmpty() {
+            assertFalse(federationStorageProvider.getProposedFederation(federationConstants, allActivations).isPresent());
```

Comment:
Nit, but I prefer to store the result of the call to the storage provider in a variable, then apply the assertion on that variable. Cleaner and easier to read